src/sentry/static/sentry/app/components/projectSelector.jsx

```diff
@@ -79,11 +85,18 @@ class ProjectSelector extends React.Component {
   }
 
   getProjects() {
-    const {organization, projects} = this.props;
-    const projectList =
-      projects || organization.projects.filter(project => project.isMember);
+    const {organization, projects, multiProjects} = this.props;
+
+    if (multiProjects) return multiProjects;
+
+    const {isSuperuser} = ConfigStore.get('user');
```

Comment:
We should probably move this into a util function since we're using it elsewhere as well.

Reply:
sgtm but i'll add this to the follow up since it's late and I'm itching to get this out